lib/tasks/migrate_advanced_search_banner_to_welcome_banner.rake

```diff
@@ -4,8 +4,21 @@
 THEME_GIT_URL ||= "https://github.com/discourse/discourse-search-banner.git"
 REQUIRED_TRANSLATION_KEYS ||= %w[search_banner.headline search_banner.subhead]
 
+desc "Run all Advanced Search Banner migration tasks"
```

Comment:
You should be able to do this instead, you can make rake tasks depend on other tasks:

```ruby
desc "Run all Advanced Search Banner migration tasks"
task "themes:advanced_search_banner:migrate_all" => [
  "themes:advanced_search_banner:1_migrate_settings_to_welcome_banner",
  "themes:advanced_search_banner:2_migrate_translations_to_welcome_banner",
  "themes:advanced_search_banner:3_exclude_and_disable"
]
```